README.md

```diff
@@ -297,9 +297,9 @@ can be used to populate a directory with template package configuration files.
 
 The file containing any policy rule implementations to be used with the _evaluator_.
 
-| Format | Scope | Default location | Default value |
-| ------ | ----- | ---------------- | ------------- |
-| Kotlin script (DSL) | Evaluator | `$ORT_CONFIG_DIR/rules.kts` | Empty (n/a) |
+| Format | Scope | Default location                | Default value |
+| ------ | ----- |---------------------------------| ------------- |
+| Kotlin script (DSL) | Evaluator | `$ORT_CONFIG_DIR/ort.rules.kts` | Empty (n/a) |
```

Comment:
I have a tendency towards `evaluator.rules.kts` as the default name.

Reply:
Fine with me, as "ort" is already part of the parent directory, and it opens the possibility to have more (non-evaluator) rules in the future.